website/docs/r/automation_software_update_configuration.html.markdown

```diff
@@ -0,0 +1,211 @@
+  sku_name            = "Basic"
+}
+resource "azurerm_automation_software_update_configuration" "example" {
+  name                    = "example"
+  resource_group_name     = azurerm_resource_group.test.name
+  automation_account_name = azurerm_automation_account.test.name
+  operating_system        = "Linux"
+  linux {
+    classification    = "Security"
+    excluded_packages = ["apt"]
+    included_packages = ["vim"]
+    reboot_setting    = "IfRequired"
+  }
+  duration = "PT2H2M2S"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Automation. Changing this forces a new Automation to be created.
+
+* `automation_account_name` - (Required) The name of the automation account in which the Schedule is created. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Automation should exist. Changing this forces a new Automation to be created.
+
+* `operating_system` - (Required) The Operating system of target machines. Possible values are `Windows` and `Linux`.
+
+---
+
+* `duration` - (Optional) Maximum time allowed for the software update configuration run. using format `PT[n]H[n]M[n]S` as per ISO8601.
+
+* `linux` - (Optional) One or more `linux` blocks as defined below.
+
+* `windows` - (Optional) One or more `windows` blocks as defined below.
+
+* `virtual_machines` - (Optional) Specifies a list of azure resource Ids of azure virtual machines.
+
+* `non_azure_computer_names` - (Optional) Specifies a list of names of non-azure machines for the software update configuration.
+
+* `targets` - (Optional) One or more `targets` blocks as defined below.
+
+* `post_task` - (Optional) One or more `post_task` blocks as defined below.
+
+* `pre_task` - (Optional) One or more `pre_task` blocks as defined below.
+
+* `schedule` - (Optional) One or more `schedule` blocks as defined below.
+
+---
+
+A `linux` block supports the following:
+
+* `classification` - (Optional) Specifies the update classifications included in the Software Update Configuration. Possible values are `Unclassified`, `Critical`, `Security` and `Other`.
+
+* `excluded_packages` - (Optional) Specifies a list of packages to excluded from the Software Update Configuration.
+
+* `included_packages` - (Optional) Specifies a list of packages to included from the Software Update Configuration.
+
+* `reboot_setting` - (Optional) Specifies the reboot settings after software update.
+
+---
+
+A `windows` block supports the following:
+
+* `classification` - (Optional) Specifies the update classification. Possible values are `Unclassified`, `Critical`, `Security`, `UpdateRollup`, `FeaturePack`, `ServicePack`, `Definition`, `Tools` and `Updates`.
+
+* `excluded_kbs` - (Optional) Specifies a list of KB numbers excluded.
+
+* `included_kbs` - (Optional) Specifies a list of KB numbers included.
```

Comment:
kbs is not clear? could we make this
```suggestion
* `excluded_knowledge_base_numbers` - (Optional) Specifies a list of knowledge base numbers excluded.

* `included_knowledge_base_numbers` - (Optional) Specifies a list of knowledge base numbers included.
```

we should avoid acronyms when possible